analyses/molecular-subtyping-HGG/README.md

```diff
@@ -0,0 +1,69 @@
+## Molecular subtyping for high-grade glioma (HGG) 
+
+This module is adapted from: [`d3b-center/OpenPedCan-analysis`](https://github.com/d3b-center/OpenPedCan-analysis).
+
+
+#### Running the analysis
+
+The analysis can be run with the following (assuming you are in the analysis directory):
+
+```
+bash run-molecular-subtyping.sh
+```
+
+### Standard of HGG molecular subtype
+
+1. If there was an _H3-3A_ K28M, _H3C2_ K28M, _H3C3_ K28M, _H3C14_ K28M, _H3-3A_ K28I, _H3C2_ K28I, _H3C3_ K28I or _H3C14_ K28I mutation -> `DMG, H3 K28`.
+2. If there was an _H3-3A_ G35V or G35R mutation -> `HGG, H3 G35`.
+3. If there was an _IDH1_ R132 mutation -> `HGG, IDH`.
+4. If high-confidence methylation classification suggested one of the subtypes above (e.g. K28) --> `DMG, H3 K28`.
+5. In `Hope-GBM-histologies_base.tsv`, column `pathology_free_text_diagnosis` contains `infant type hemispheric glioma` or `cns_methylation_subclass` == "IHG" -> `IHG`
+    1. If there was a _NTRK_ fusion -> `IHG, NTRK-altered`
+    2. If there was a _ROS1_ fusion -> `IHG, ROS1-altered`
+    3. If there was a _ALK_ fusion -> `IHG, ALK-altered`
+    4. If there was a _MET_ fusion -> `IHG, MET-altered`
+    5. If there was no fusion -> `IHG, To be classified` based on IHG methylation classification and sample clinical report in the `pathology_diagnosis_free_text` stated as `infant type hemispheric glioma`.
+6. Either in `Hope-GBM-histologies_base.tsv`, column `pathology_free_text_diagnosis` contains `malignant pxa", "pleomorphic xanthoastrocytoma`, `anaplastic pleomorphic xanthoastrocytoma with braf p.val600glu mutation, who grade iii`, `anaplastic pleomorphic xanthoastrocytoma, who grade 3` 
+**or** 
+`methylation_subclass == "PXA"`, 
+**and** 
+have _BRAF_ mutation or other MAPK pathway gene alteration, combined with homozygous deletion of _CDKN2A_ and/or _CDKN2B_. -> "PXA".
+7. All samples that did not meet any of these criteria were marked as `HGG, H3 wildtype` (i.e. there was no canonical histone variant the DNA or methylation classification subtype). 
```

Comment:
```suggestion
7. All samples that did not meet any of these criteria were marked as `HGG, H3 wildtype` (i.e. there was no canonical variant the DNA or methylation classification subtype). 
```
one more update bc this could also refer to IDH, but otherwise good to go!

Reply:
Nice catch! Change has been made and I will merge it into main branch